.github/workflows/pr-check.yaml

```diff
@@ -20,10 +20,66 @@ name: pr-check
 on: [pull_request]
 
 jobs:
-  pr-check-job:
+  
+  build:
+    name: Build
     strategy:
       matrix:
         target: [windows-2022, macos-14, ubuntu-20.04]
     uses: redhat-developer/podman-desktop-redhat-account-ext/.github/workflows/build.yaml@main
     with:
-      runs-on: ${{ matrix.target }}
\ No newline at end of file
+      runs-on: ${{ matrix.target }}
+  
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        target: [windows-2022, macos-14, ubuntu-20.04]
+      
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Get yarn cache directory path (Windows)
+        if: ${{ matrix.os=='windows-2022' }}
+        id: yarn-cache-dir-path-windows
+        run: echo "dir=$(yarn cache dir)" >> ${env:GITHUB_OUTPUT}
+
+      - name: Get yarn cache directory path (mac/Linux)
+        if: ${{ matrix.os=='ubuntu-22.04'}}
```

Comment:
mismatched matrix property -> target vs. os. here you are checking ubuntu 22.04 but only 20.04 is available.

Reply:
Good catch! I was writing upper part and the copy/paste cache part form podman-desktop pr-check.